src/Entity/User/OAuthToken.php

```diff
@@ -87,32 +88,18 @@ public function isType(string $type): bool
         return $this->type() === $type;
     }
 
-    public function accessToken(): string
+    public function accessToken(ClientRegistry $oauth): string
```

Comment:
this will be replaced in future with custom service to avoid coupling with `OAuth2ClientBundle`